src/main.ts

```diff
@@ -151,8 +153,26 @@ async function findAllActions(
     const content = await getActionFile(client, repo)
     if (content && content.name !== '') {
       core.info(
-        `Found action file in repository: ${repo.name} with filename [${content.name}] download url [${content.downloadUrl}]`
+        `Found action file in repository: ${repo.name} with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is ${repo.visibility}`
```

Comment:
Let's follow the convention here like in the rest of the repo:
```suggestion
        `Found action file in repository: [${repo.name}] with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is [${repo.visibility}]`
```

Reply:
Updated with changes